### PR TITLE
Src/add param class

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/Param.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.jalasoft.jfc;
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/Param.java]: add copyright
add class documentation
/**
 *
 *  Class description.
 *
 * @version 1.0 13 Dic 2019
 *
 * @author FirstName LastName
 */

File: src/main/java/com/jalasoft/jfc/Param.java [hunk 2]
```diff
@@ -1,8 +1,27 @@
...
+* @version 1.0  Dic 2019
+*
+* @author Alan Escalera
+*/
 public class Param {
-  /**
-   * This Class is the class father.
-   */
+
   private String InputPathFile;   //Input path file
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/Param.java]: piriod at end of each coment

File: src/main/java/com/jalasoft/jfc/Param.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ *  Copyright (c) 2019 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not
+ *  disclose such Confidential Information and shall use it only in
+ *  accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ * /
```
Comment (Enrique-C) [inline src/main/java/com/jalasoft/jfc/Param.java]: This line is not necesary

File: src/main/java/com/jalasoft/jfc/Param.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+package com.jalasoft.jfc;
+
+/**
+*
+*  Param Class.
```
Comment (Enrique-C) [inline src/main/java/com/jalasoft/jfc/Param.java]: More information

File: src/main/java/com/jalasoft/jfc/Param.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+*
+* @version 1.0  Dic 2019
+*
+* @author Alan Escalera
+*/
+public class Param {
+
+  private String InputPathFile;   //Input path file.
+  private String OutputPathFile;  //Output path file.
```
Comment (Enrique-C) [inline src/main/java/com/jalasoft/jfc/Param.java]: These lines need more information

**Review by Enrique-C — APPROVED** [review]

**Review by Enrique-C — APPROVED** [review]

**Pull request merged**